guides/doc-Upgrading_and_Updating/topics/upgrading_satellite_server_prerequisites.adoc

```diff
@@ -19,6 +19,7 @@ Cloning is the recommended method because that prevents them being overwritten i
 To confirm if a template has been edited, you can view its *History* before you upgrade or view the changes in the audit log after an upgrade.
 In the {ProjectWebUI}, navigate to *Monitor* > *Audits* and search for the template to see a record of changes made.
 If you use the export method, restore your changes by comparing the exported template and the default template, manually applying your changes.
+* If you previously upgraded {Project} from 6.7 or an earlier version, or if the `/var/lib/pgsql` directory has any content, ensure it is empty before proceeding. {Project} {ProjectVersionPrevious}'s migrated `postgres` database content will be stored in this directory.
```

Comment:
`6.7` is Satellite-only. Please check if this file is only included in builds for Satellite. If it also shows for Foreman, please either remove the version number or encapsulate in within `ifdef::satellite[]` and add an appropriate sentence for Foreman please.

Also, please stick to one sentence per line.